apps/apps.c

```diff
@@ -686,18 +693,45 @@ X509_CRL *load_crl(const char *infile, int format)
     } else if (format == FORMAT_PEM) {
         x = PEM_read_bio_X509_CRL(in, NULL, NULL, NULL);
     } else {
-        BIO_printf(bio_err, "bad input format specified for input crl\n");
+        print_format_error(format, OPT_FMT_PEMDER);
         goto end;
     }
-    if (x == NULL) {
-        BIO_printf(bio_err, "unable to load CRL\n");
+
+ end:
```

Comment:
you should move this label after the `if (x == NULL && desc != NULL)`

Reply:
This has already been done in my last commit of Friday: d918db0